requirements.txt

```diff
@@ -1,9 +1,35 @@
+astroid==1.5.3
```

Comment:
What are all these packages for?

Reply:
- Most of these packages are dependencies of packages used within the code. 

- Other packages are development dependencies e.g. `pylint`, `coveralls`, `coverage`, `nose`, `pycodestyle` and `rope`.

- So when we run `pip freeze`, they are also added to the requirements file. Is there a way we can avoid this, kindly let us know.

Reply:
pylint, pycodestyle, rope are not being used in this project. If you removed all the packages from the env and installed the ones in the master/requirements.txt + pep8, check it runs successfully